package.json

```diff
@@ -52,5 +53,8 @@
     "rollup-plugin-peer-deps-external": "^2.2.4",
     "ts-jest": "^29.0.2",
     "typescript": "^4.5.5"
+  },
+  "dependencies": {
+    "cookie": "git+https://github.com/jshttp/cookie.git#663c9aeb85c9e046ff2ad2cdec631afdd40a7965"
```

Comment:
Is there you need a version that exactly the commit 663c9aeb85c9e046ff2ad2cdec631afdd40a7965?
Why don't just add `cookie` with version from NPM registry?

Reply:
If we use the latest version, the package owner can release a new version and insert malicious code into our customers' Cloudflare accounts without us noticing. I know it is an edge case, but it is better to use fixed versions. We do the same for GitHub actions. ([example](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/blob/main/.github/workflows/release.yml#L35))

Reply:
The commit `663c9aeb85c9e046ff2ad2cdec631afdd40a7965` is the latest commit right now

Reply:
You can fix the concrete version from npm, they deny to unpublish version of used package after 72 hours after publishing https://docs.npmjs.com/policies/unpublish

Reply:
Got it. Fixed in the next commit @ilfa @sshelomentsev